corehq/motech/dhis2/forms.py

```diff
@@ -45,39 +47,23 @@ class DataSetMapForm(forms.ModelForm):
     )
     data_set_id = forms.CharField(
         label=_('DataSetID'),
-        help_text=_('Set DataSetID if this UCR adds values to an existing '
-                    'DHIS2 DataSet'),
+        help_text=_('The DHIS2 DataSet ID to which this DataSet adds value.'),
         validators=[RegexValidator(DHIS2_UID_RE, DHIS2_UID_MESSAGE)],
-        required=False,
+        required=True,
```

Comment:
If the code is going to expect this always to have a value, we will need a plan for existing instances where this has no value.

* See if there are any DataSets on Staging and Prod where this is NULL
* Evaluate them to see if they are in use, and how to proceed.

I suspect this always has a value for real integrations. If we can resolve any instances that don't have a value, we can change the model to `null=False, empty=False`.

Reply:
Yes! Thanks for reminder, I'll quickly make a note to check that!